tests/test_cli.py

```diff
@@ -19,7 +19,7 @@ def initdir(self, tmpdir):
     def s3_address(self, request):
         urls = {
             "0.1": "https://s3.embassy.ebi.ac.uk/idr/zarr/v0.1/6001240.zarr",
-            "0.3": "https://minio-dev.openmicroscopy.org/idr/v0.3/idr0077-valuchova-flowerlightsheet/9836842.zarr"
+            "0.3": "https://minio-dev.openmicroscopy.org/idr/v0.3/idr0077-valuchova-flowerlightsheet/9836842.zarr",
```

Comment:
I am not sure what's going on here; for some reason this undoes the changes in my previous commit:
https://github.com/ome/ome-zarr-py/pull/89/commits/a6ea341cdc27d7afbc82b7024004630e1461e072